testing/rspec/spec/unit/linters/unique_scenario_names_linter_spec.rb

```diff
@@ -0,0 +1,261 @@
+RSpec.describe CukeLinter::UniqueScenarioNamesLinter do
+  let(:linter) { described_class.new }
+  subject { linter }
+
+  it_should_behave_like 'a linter at the unit level'
+
+  it 'has a name' do
+    expect(subject.name).to eq('UniqueScenarioNamesLinter')
+  end
+
+  before do
+    described_class.instance_variable_set(:@scenario_names, {})
+  end
+
+  describe 'linting' do
+
+    context 'with the same scenario names across different feature files' do
```

Comment:
Missing case:

Same scenario name (non outline) and scenario name generated from outline.

Reply:
It's on a line 108